### PR TITLE
Add the `latex.x86_64` image for building the PDF docs on Buildkite

### PR DESCRIPTION
**Pull request by DilumAluthge**
Let's see if `--no-install-recommends` helps us keep this under 5GB.

### CODE REVIEW

**Comment by DilumAluthge** [comment]
@mortenpi This is based on https://github.com/JuliaDocs/DocumenterLaTeX.jl/blob/master/docker/Dockerfile.

**Comment by mortenpi** [comment]
I think we can actually omit the fonts (i.e. `font-lato` and the Roboto Mono download), since we switched away from using those fonts in the PDF: https://github.com/JuliaDocs/Documenter.jl/pull/1066

**Comment by DilumAluthge** [comment]
So we can remove all of the Google Fonts stuffv

**Comment by DilumAluthge** [comment]
Can I also remove the call to `fc-cache -fv`?

File: linux/latex.jl [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "locales",
+    "localepurge",
+]
+
+artifact_hash, tarball_path, = debootstrap(arch, image; archive, packages) do rootfs, chroot_ENV
+    my_chroot(args...) = root_chroot(rootfs, "bash", "-eu", "-o", "pipefail", "-c", args...; ENV=chroot_ENV)
+
+    additional_packages = String[
+        "curl",
+        "fontconfig",
```
Comment (mortenpi) [inline linux/latex.jl:21]: I reckon this can go as well.

**Review by mortenpi — APPROVED** [review]
:crossed_fingers:

**Comment by DilumAluthge** [comment]
Alright, fingers crossed that this ends up being under 5GB once compressed. Otherwise, we won't be able to upload it to GitHub Releases.

**Comment by mortenpi** [comment]
For what it's worth, the Docker image appears to be ~500MB: https://hub.docker.com/layers/juliadocs/documenter-latex/0.1/images/sha256-3ec7a97f7d3ff575879195c6d5a011f350ada196ed82fb3483ce6a8984f21588?context=explore

**Pull request merged**